### PR TITLE
[INLONG-9883][Manager] Add unit test of HeartbeatController

### LINKED ISSUE #9883: [Improve][Manager] Add unit test of HeartbeatController
### Description

What happened
add ut HeartbeatController

What you expected to happen
add ut HeartbeatController

How to reproduce
add ut HeartbeatController

Environment
add ut HeartbeatController

InLong Component
inlong-manager

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by XiaoYou201**
### Prepare a Pull Request

- Title Example: [INLONG-9883][Manager] Add unit test of HeartbeatController

- Fixes #9883 

### Motivation

Complete the pr #9887 

### Modifications

Add HeartbeatController, Modify assert statement condition

### CODE REVIEW

**Comment by fuweng11** [comment]
This UT cannot verify whether the API's functionality meets expectations.Please refer to StreamSourceServiceTest.testGetAndUpdate to verify whether the actual functionality of the API is effective.

**Review by fuweng11 — DISMISSED** [review]

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+public class HeartbeatControllerTest extends WebBaseTest {
+
+    @Autowired
+    private HeartbeatService heartbeatService;
+
+    @InjectMocks
+    private HeartbeatController heartbeatController;
+
+    private static final String TEST_PWD = "test_#$%%Y@UI$123";
+    private static final String INSTANCE_IP = "192.168.1.3";
```
Comment (fuweng11) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java:59]: private static final String INSTANCE_IP = "127.0.0.1";

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    private static final String COMPONENT_TYPE = "DATAPROXY";
+    private static final String INLONG_GROUP_ID_1 = "test_group_heartbeat_1";
+    private static final String INLONG_GROUP_ID_2 = "test_group_heartbeat_2";
+    private static final String INLONG_STREAM_ID_1 = "test_stream_heartbeat_1";
+    private static final String INLONG_STREAM_ID_2 = "test_stream_heartbeat_2";
+    private static final String STATUS = "enabled";
+    private static final String CLUSTER_NAME = "test_cluster";
+    private static final String CLUSTER_TAG = "test_cluster";
+    private static final String EXT_TAG = "test_ext_tag";
+    private static final String PORT = "12345";
```
Comment (fuweng11) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java:69]: private static final String PORT = "46801";

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java (hunk 1 above)
Comment (XiaoYou201) [reply inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java]: fixed it

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java (hunk 2 above)
Comment (XiaoYou201) [reply inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/HeartbeatControllerTest.java]: fixed it

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**